department-of-veterans-affairs/caseflow
# APPEALS-9132 and APPEALS-9133

Resolves [APPEALS-9132](https://vajira.max.gov/browse/APPEALS-9132 ) and [APPEALS-9133](https://vajira.max.gov/browse/APPEALS-9133)



### Description
Makes SSN and file number searchable for reviews

### Acceptance Criteria
- [x] Veteran SSN and File Number are searchable
- [x] Searching for participant ID and name still work
- [x] SSN and File Number should only be searchable if the Feature Toggle decision_review_queue_ssn_column is enabled

### Testing Plan
1. Generate some data. Open a rails console and do `100.times.map { FactoryBot.create(:higher_level_review_vha_task) }
 `
2. In the same console run the command to enable the feature toggle: `FeatureToggle.enable!(:decision_review_queue_ssn_column)`
3. Go to decision_reviews/vha (log in as the user ACBAUERVVHAH)
4. To get some numbers to use for a search, in a rails console, do: `DecisionReviewTask.last.appeal.veteran`. You'll need the ssn and file_number.
5. Type a veteran's ssn, and the associated review should appear
6. Clear search and type a veteran's file number, and the associated review should appear
7. Also search by participant ID and name to make sure they still work
8. Disable the feature toggle by running this command in a rails console: `FeatureToggle.disable!(:decision_review_queue_ssn_column)`
9. repeat step 7
